metadata/algolia/jobs.xml

```diff
@@ -130,7 +130,6 @@ Relies on B2C Delta Exports to determine which products have been changed since
                 <parameters>
                     <parameter name="consumer" job-parameter-ref="consumer"/>
                     <parameter name="deltaExportJobName" job-parameter-ref="deltaExportJobName"/>
-                    <parameter name="fullRecordUpdate">true</parameter>
```

Comment:
`fullRecordUpdate` should be replaced with `indexingMethod`, not removed. See #74 .

Reply:
Since it's an enum with a default value, it works fine without

Reply:
It should still be explicitly added to show the intention. And also because the job definition is incomplete otherwise.

Reply:
alright